clustoapi/apps/attribute.py

```diff
@@ -21,6 +21,61 @@
 app.config['source_module'] = __name__
 
 
+def _write_attrs(method, name, **kwargs):
+    """
+Helper method for reduced code between POST and PUT.
+Returns a response for the methods calling it.
+"""
+    if method == 'set_attr':
+        code = 200
+    if method == 'add_attr':
+        code = 201
+    else:
```

Comment:
I mean.. the only thing we're doing before bailing is comparing a primitive twice. I really don't think performance would be an issue and it would only make the block have more code.